src/features/identityCheck/api.ts

```diff
@@ -27,8 +27,8 @@ export function useIdentificationUrl() {
         setIdentificationUrl(data.identificationUrl)
       } catch (err) {
         const error = (err as ApiError)?.content.code
-        if (error === 'IDCHECK_ALREADY_PROCESSED') navigate('IdentityCheckPending')
-        navigate('IdentityCheckUnavailable', { withDMS: false })
+        if (error === 'IDCHECK_ALREADY_PROCESSED') return navigate('IdentityCheckPending')
```

Comment:
```suggestion
        if (error === 'IDCHECK_ALREADY_PROCESSED') {
          navigate('IdentityCheckPending')
        } else {
          navigate('IdentityCheckUnavailable', { withDMS: false })
        }
```

Dimitri faisait remarquer à juste titre que `return void` est incorrect.